README.md

```diff
@@ -100,7 +101,7 @@ Which one of the following concepts does this best illustrate?
 What is the `ui-sref` directive, and how is it used?
 
 ```text
-Your answer goes here...
+It is the angular equivalent of href. It is used for linking in angular.
```

Comment:
Specifically it is for loading a particular state. It's not like traditional linking because we are not loading an entirely new page.